### ISSUE TITLE
Cypress `v11.0.0` Broken with `experimentalSessionAndOrigin: true`

### ISSUE THREAD

**Issue by stychu** [cypress-io/cypress#24611]
### Current behavior

I wanted to upgrade to cypress `v11` from `v10.4.0`. After bumping the version Cypress is broken.
Figuring out whats wrong I found that the `experimentalSessionAndOrigin: true` leads to this error which prevents cypress from running the tests

When I disable `experimentalSessionAndOrigin` tests seem to work but obviously don't pass as I use session and origin

```js
Error: Webpack Compilation Error
./cypress/tests/e2e/_userLogin.cy.ts
Module build failed (from /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/lib/cross-origin-callback-loader.js):
Error: Cannot find module '/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@babel/parser/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at Watching.handle [as handler] (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:230:23)
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:99:9
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:154:20)
    at Watching._done (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:98:28)
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:73:19
    at Compiler.emitRecords (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:499:39)
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:54:20
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:485:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:154:20)
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:482:27
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:2818:7
    at done (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:3522:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:154:20)
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:464:33
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /Users/..../Library/Caches/Cypress/11.0.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:61:14
    at FSReqCallback.oncomplete (fs.js:179:23)
```

### experimentalSessionAndOrigin: true
<img width="1636" alt="Screenshot 2022-11-09 at 13 08 04" src="https://user-images.githubusercontent.com/9383657/200827005-1cc21eb8-ee50-4ccb-a9e9-7fb50e2fa2a3.png">

### experimentalSessionAndOrigin: false
<img width="1630" alt="Screenshot 2022-11-09 at 13 08 16" src="https://user-images.githubusercontent.com/9383657/200827011-697af3d1-31bc-45b1-b022-15039b3b29bb.png">

### Desired behavior

Cypress successfully runs the tests as normal

### Test code to reproduce

....

### Cypress Version

    "cypress": "11.0.0"

### Node version

v14.18.1

### Operating System

macOS BigSur 11.7

**Comment by emilyrohrbough** [comment]
We are actively investigating.

**Comment by mjhenkes** [comment]
Released in [11.0.1](https://on.cypress.io/changelog#11-0-1).

This comment thread has been locked. If you are still experiencing this issue after upgrading to
Cypress v11.0.1, please open a new issue.

**Comment by karlhorky** [comment]
Thanks for the quick reaction and release @ryanthemanuel @emilyrohrbough @mjhenkes  @ZachJW34 ! 🙌
